unit-tests/live/d500/test-get-set-calib-config-table-api.py

```diff
@@ -0,0 +1,242 @@
+# License: Apache 2.0. See LICENSE file in root directory.
+# Copyright(c) 2024 Intel Corporation. All Rights Reserved.
+
+# test:donotrun
+
+import pyrealsense2 as rs
+from rspy import test, log
+from rspy import tests_wrapper as tw
+import json
+
+#############################################################################################
+# Tests
+#############################################################################################
+
+test.start("Verify auto calibrated device extension")
+ctx = rs.context()
+dev = ctx.query_devices()[0]
+ac_dev = dev.as_auto_calibrated_device()
+
+test.finish()
+
+#############################################################################################
+
+tw.start_wrapper(dev)
+
+
+test.start("Writing calibration config, then reading and comparing calibration config JSONs")
+
+# save current calibration config json in order to restore it at the end
+original_calib_config = ac_dev.get_calibration_config()
```

Comment:
Not sure we want to write calibration tables..
If the test stop in the middle we will get a very bad calibration table on the CI machines no?
How do you think we can make this more robust?

Reply:
Let's decide this offline.